### ISSUE TITLE
Analytic calculation of cooling power

### ISSUE THREAD

**Issue by bvlehm** [stanford-physics108-2015/cryo#4]
@marxwillia @zmwangx @rexgarland

**Comment by marxwillia** [comment]
This link provides an example of what we want to do.  See exercise 2 on Slide 78:

http://hep.ph.liv.ac.uk/~hock/Teaching/9-magnetic-refrigerator.pdf

**Comment by bvlehm** [comment]
We have made a preliminary calculation for CPA. See the Mathematica notebook at (Dropbox)/Mathematica/temperature_calculation_CPA.nb. Unfortunately, we have slightly different data available for FAA, so we will need to redesign the calculation slightly.

Also note the heat leak calculation at (Dropbox)/Mathematica/temperature_calculation_CPA.nb. We need to think a little more about the heat sinking.

**Comment by zmwangx** [comment]
Ben's notebook is also in the repo as `mma/temperature_calculation_cpa.nb`. I made one rather cosmetic change:
![](http://i.imgur.com/4Xu6SLm.png)
to
![](http://i.imgur.com/sT7SSUb.png)
, and I stripped the notebook history; no more changes.

(By the way, I'm using a lowercase filename because case can cause headaches with git — git is case-sensitive while HFS+ is not. Uppercase isn't likely to cause trouble in our case, but I believe using lowercase filenames — especially in git repos is a good practice)

---

The heat leak calculation is yet to be reviewed, hence not committed in.
